tensorflow/lite/tools/delegates/delegate_provider.cc

```diff
@@ -32,6 +32,18 @@ void ProvidedDelegateList::AppendCmdlineFlags(std::vector<Flag>* flags) const {
   }
 }
 
+void ProvidedDelegateList::RemoveCmdlineFlag(std::vector<Flag>& flags,
```

Comment:
Move this function as part of class "Flags" considering the functionality has nothing to do with ProvidedDelegateList?

Reply:
The functionality is provided in `ProvidedDelegateList` class as the counterpart of `AppendCmdlineFlags` method (already in the class). The `Flags` class currently serves only to parse/print:  
```
class Flags {
 public:
  static bool Parse(int* argc, const char** argv,
                    const std::vector<Flag>& flag_list);

  static std::string Usage(const std::string& cmdline,
                           const std::vector<Flag>& flag_list);

  static std::string ArgsToString(int argc, const char** argv);
};
```

Reply:
This makes sense. Btw, to be consistent with the declaration of AppendCmdlineFlags, change "std::vector<Flag>& flags" to "std::vector<Flag>* flags" or vice versa?

Reply:
Make sense. For consistency replaced the pointer to reference in `AppendCmdlineFlags`. The pointer is not necessary. Moreover this way it is not necessary to check the pointer validity (i.e. != nullptr).

Reply:
Great! Thx!